crowdin_api/api_resources/application/resource.py

```diff
@@ -0,0 +1,56 @@
+from crowdin_api.parser import dumps
+
+from crowdin_api.api_resources.abstract.resources import BaseResource
+
+
+class ApplicationResource(BaseResource):
+    """
+    Resource for Applications.
+
+    Link to documentation:
+    https://developer.crowdin.com/api/v2/#tag/Applications
+
+    Link to documentation for enterprise:
+    https://developer.crowdin.com/enterprise/api/v2/#tag/Applications
+    """
+
+    def get_application_path(self, applicationIdentifier: str, path: str):
+        if applicationIdentifier and path:
+            return f"applications/{applicationIdentifier}/api/{path}"
+
+    def get_application_data(self, applicationIdentifier: str, path: str):
+
```

Comment:
please include comments for each API endpoint

for example, as it was in this PR - https://github.com/crowdin/crowdin-api-client-python/pull/134/files